src/CameraSensor.cc

```diff
@@ -238,6 +236,24 @@ bool CameraSensor::CreateCamera()
     this->dataPtr->saveImage = true;
   }
 
+  auto *cameraSdfCopy = const_cast<sdf::Camera *>(cameraSdf);
```

Comment:
Is const_cast safe here? Maybe we can just declare `cameraSdf` and non-const?

Reply:
yeah we can do that sounds good.